packages/shared/src/telemetry/events/theme-usage.ts

```diff
@@ -0,0 +1,83 @@
+import type { Appearance, BaseTheme, TelemetryEventRaw } from '@clerk/types';
+
+const EVENT_THEME_USAGE = 'THEME_USAGE';
+const EVENT_SAMPLING_RATE = 0.1;
```

Comment:
For launch let's increase the sample rate to 100% - this will give us much more accurate picture of how often it's being used. Events are still throttled via localstorage so we shouldn't have duplicates and the volume won't be unmanageable. 

```suggestion
const EVENT_SAMPLING_RATE = 1;
```